docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-db-audit-logs-exporter.md

```diff
@@ -1,4 +1,4 @@
-<!---
+---
```

Comment:
Keep this commented. We are not releasing CLI with this release as CLI commands are getting refactored.